.NET/Microsoft.Recognizers.Definitions.Common/Portuguese/DateTimeDefinitions.cs

```diff
@@ -86,7 +87,8 @@ public static class DateTimeDefinitions
       public const string SpecialDayRegex = @"\b((d?o\s+)?(dia\s+antes\s+de\s+ontem|antes\s+de\s+ontem|anteontem)|((d?o\s+)?(dia\s+|depois\s+|dia\s+depois\s+)?de\s+amanh[aã])|(o\s)?dia\s+seguinte|(o\s)?pr[oó]ximo\s+dia|(o\s+)?[uú]ltimo\s+dia|ontem|amanh[ãa]|hoje)|(do\s+dia$)\b";
       public const string SpecialDayWithNumRegex = @"^[.]";
       public const string ForTheRegex = @".^";
-      public const string WeekDayAndDayOfMonthRegex = @".^";
+      public static readonly string FlexibleDayRegex = $@"(?<DayOfMonth>([A-Za-z]+\s)?({WrittenDayRegex}|{DayRegex}))";

```

Comment:
No need to specify "A-Z" as for perf pre-processing guarantees capital letters won't reach the regex.

Reply:
removed